main/wallet-api.md

```diff
@@ -0,0 +1,206 @@
+  - `payment`
+    - **Object**. A payment object, which is deposited into the specified purse.
+- **Optional Arguments:**
+  - None.
+
+### `getPurses()`
+- **Function:**
+  - Returns all purse objects and their petnames in the wallet. They are returned as an array of [petname, purseObject] arrays which can be turned into a `Map` (`new Map(getPurses())`)
+- **Required Arguments:**
+  - None.
+- **Optional Arguments:**
+  - None.
+
+### `getPurse(pursePetname)`
+- **Function:**
+  - Returns the `purse` object with the given petname
+- **Required Arguments:**
+  - `pursePetName`  
+    - **String**. A purse's petname.
+- **Optional Arguments:**
+  - None.
+- **Errors**
+  - Throws an error if there is no purse with the given petname.
+
+### `getPurseIssuer(pursePetname)`
+- **Function:**
+  - Given a purse's petname, returns the `issuer` object associated
+    with the purse.
+- **Required Arguments:**
+  - **String**. `petname`, a purse's petname
+- **Optional Arguments:**
+  - None.
+- **Errors**
+  - Throws and error if given an invalid purse petname. 
+
+### `makeEmptyPurse(issuerPetname, pursePetname)`
+- **Function:**
+  - Creates and returns a new, empty, purse object with a petname specified by the command. 
+- **Required Arguments:**
+  - `issuerPetname`
+    - **String**. The petname you've assigned to the `issuer` the new purse uses.
+  - `pursePetname`
+    - **String**. The petname assigned to the new purse.	
+- **Optional Arguments:**
+ - None.
+- **Errors:**
+  - There is already a purse in this wallet with the name of `pursePetname`'s value.   
+
+
+## Issuer API Methods
+
+### `addIssuer()`
+- **Function:**
+  -  Assigns the given petname to a given`issuer` object and adds the issuer to the wallet. 
+- **Required Arguments:**
+  - `issuerPetname`
+    - **String**. The petname assigned to the `issuer` object.
+  - `issuer`
+    - **Object**. An `issuer` object.
+- **Optional Arguments:**
+  - `brandRegKey`
+    - **String**. A key in the Registry, whose value is the `brand` object associated with the issuer.
+
+### `getIssuers()`
+- **Function:**
+  - Returns all issuers associated with the wallet and their petnames as an array of arrays of the format `[[issuerPetname, issuer], ...]`. You can use it to make a new map of petnames to issuers by doing `new Map(getIssuer())` 
+- **Required Arguments:**
+  - None.
+- **Optional Arguments:**
+  - None.
+
+### `getIssuerNames(issuer)`
+- **Function:**
+  - Returns the petname and brandRegKey of a particular issuer object
+- **Required Arguments:**
+  - `issuer`
+    - An issuer object
+- **Optional Arguments:**
+  - None.
+
+## Offer API Methods
+
+### `acceptOffer()`
+- **Function:**
+  - Changes the specified offer's status to "accepted".
+- **Required Arguments:**
+  - `id`
+    - **String**: An offer id.
+- **Optional Arguments:**
+  - None.
+- **Errors:**
+  - If the offer has already been resolved or rejected. 
+
+### `addOffer()`  
```

Comment:
This really just adds a proposal that the user may approve to actually make an offer to the contract invitation.